lib/Core/Memory.cpp

```diff
@@ -1046,22 +1032,23 @@ std::string PersistentState::getAllocInfo(ref<Expr> offset,
   getObject()->getAllocInfo(tmp);
   msg << tmp << "\n";
 
-  return infoStr;
+  return msg.str();
```

Comment:
Wait, the other way isn't right?

Reply:
Yeah, I ran into an issue where it's sometimes not "flushed". So it's mostly right, unless you don't end with a newline I guess.